src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/QueryGenerators/SqlQueryGenerator.cs

```diff
@@ -381,6 +389,10 @@ public object VisitTable(TableExpression tableExpression, SearchOptions context)
                 case TableExpressionKind.Include:
                     var includeExpression = (IncludeExpression)tableExpression.NormalizedPredicate;
 
+                    _includeCtes = _includeCtes ?? new List<string>();
```

Comment:
This one is actually a list of ids, same for the _includeFromCtes.
can we have this in the var name to make it easier to read

Reply:
I'll rename to _includeCteNames - I think Name is better than Id in case of cte